### DISCUSSION TITLE
Failed to verify student information but showed success and could not be resubmitted for verification

### DISCUSSION THREAD

**Discussion by wyf111** [community/community#122492] [GitHub Education]
### Hi everyone,

I have verified the student information on github and it is shown to be available. But the email said it did not complete the personal information. So what do I do next so I can use my equity.

Email：
Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
<img width="227" alt="1714975830013" src="https://github.com/community/community/assets/38495966/62549a82-38ab-40fc-b533-4f38c55a0303">

**Comment by davevad93** [comment]
Hi @wyf111 ,

Technically, that message states that your benefits are already available, so I think you don't have to do anything because I doubt that GitHub would have accepted your request if some of your personal information were incorrect or incomplete, besides, as far as I can see you have the PRO badge, so that confirms that everything is ok. In any case if you want to be 100% sure, contact with [GiHub Education Support](https://support.github.com/request/education)
